crates/evm/src/coverage/mod.rs

```diff
@@ -115,7 +115,7 @@ impl CoverageReport {
 
 /// A collection of [HitMap]s
 #[derive(Default, Clone, Debug)]
-pub struct HitMaps(pub HashMap<H256, HitMap>);
+pub struct HitMaps(pub HashMap<B256, HitMap>);
```

Comment:
why are we using B256 now?
isn't there an alias for H256? 
this can be super confusing

Reply:
https://docs.rs/alloy-primitives/latest/alloy_primitives/aliases/type.B256.html

Reply:
H stands for Hash which we moved away from months ago :)